### PR TITLE
Updated to use yaml file for ken

### PR DESCRIPTION
**Pull request by jimni1222**
Updated config file format to yaml from toml.

Also i upgraded rosetta-klaytn version to release this feature.

### CODE REVIEW

File: klaytn/ken.yaml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+syncmode: full
```
Comment (kjhman21) [inline klaytn/ken.yaml:1]:
```suggestion
syncmode: archive
```

Comment (jimni1222) [reply klaytn/ken.yaml:1]: I think syncmode and gcmode is different.
```
	SyncModeFlag    = TextMarshalerFlag{
		Name:   "syncmode",
		Usage:  `Blockchain sync mode ("full" or "snap")`,
		Value:  &defaultSyncMode,
		EnvVar: "KLAYTN_SYNCMODE",
	}
	GCModeFlag = cli.StringFlag{
		Name:   "gcmode",
		Usage:  `Blockchain garbage collection mode ("full", "archive")`,
		Value:  "full",
		EnvVar: "KLAYTN_GCMODE",
	}
```

Comment (jimni1222) [reply klaytn/ken.yaml:1]: You can find gc mode flag in https://github.com/klaytn/rosetta-klaytn/pull/130/files#diff-8183fe370f9d1db240e6998e8260670d0a1b7c5e8c651eda2b3e8dae8d7e1ab2R111

Comment (kjhman21) [reply klaytn/ken.yaml:1]: Okay got it.

**Review by kjhman21 — APPROVED** [review]

**Pull request merged**